client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
+
+.PinkHeader {
+    background-image: var(--profile-banner);
+    background-repeat:no-repeat;
+    background-size: cover;
+    width: 1440px;
```

Comment:
Why does this have an explicit width? We'll want it to take up the width of the page, which may be more or less than 1440px